### DISCUSSION TITLE
忽然发现FIrefox可以改accept-language了。

### DISCUSSION THREAD

**Discussion by daiaji** [jiacai2050/gooreplacer4chrome#137] [Q&A]
以前不可以改来着，现在把accept-language改成ja，就能逛DMM了。

**Comment by jiacai2050** [comment]
好奇你是怎么发现的

**Comment by daiaji** [comment]
我忽然发现我的网页变成家鄉の文字了，我寻思我也梅有那么狗罕见啊，我不是刘路，不懂日本话啊，不可能设置默认语言为日文啊 。

找了一圈发现请求头被修改了。
我想我之前折腾这玩意貌似是失效的。

难道…

关掉扩展发现一切都回来了。
后知后觉发现能改accept-language了？

dmm页面验证？
以前上网冲浪别人告诉我默认日文就能过验证。
属于是老网民の豆知识了。

现在验证代码也没改嘛。
老jp人了。

话说ch系能改吗？
